### ISSUE TITLE
New Bandwidth Dashboard API URLs

### ISSUE THREAD

**Issue by hopewise** [Bandwidth/ruby-bandwidth-iris#52]
Hello

According to this doc https://api.test.inetwork.com/apidocs/navsidebar/newURLs.pdf
How can we configure the gem to use the new API URL?

Also, is there anything else we need to change in our code to adopt with the new URL? we are currently using v 2.6.0

**Comment by ajrice6713** [comment]
Hey @hopewise 

Those URLs were updated in PR #35 to reflect the migration from the `inetwork` urls to the new `dashboard` url

The prod API URL for the `dashboard is dashboard.bandwidth.com/api`

the test environment is `test.dashboard.bandwidth.com/api`

As long as you are using the latest version of the `ruby-bandwidth-iris` client you should have no issues 

The `api.inetwork` urls were decomissioned months ago so any api calls to them would not resolve. 

Let me know if you have any questions!

**Comment by hopewise** [comment]
Oh, thanks, so it seems there isn't any change regarding the request or response format, only the URL need to be changed using the global_options, correct?

**Comment by ajrice6713** [comment]
@hopewise thats correct! Be sure to set the api endpoint to `https://dashboard.bandwidth.com` in the `client.rb` file and the SDK should function as expected - no changes to any request/response format that I am aware of from 2.6.0 unless a breaking change was introduced to the API itself - but we keep those to a minimum so there shouldnt be any major issues

**Comment by hopewise** [comment]
Thanks, but there isn't any request/response format change from the the API itself other than the URL?

**Comment by ajrice6713** [comment]
`V2.6.0` was released on June 09, 2020 - Looking back through all of the closed PRs it looks like nothing was changed in that SDK that would have reflected a fix for a breaking change to the API 

When the change was made from the inetwork url to the dashboard url there were no changes to the request/response bodies - so you shouldn't have any issues using the SDK and just changing that URL in the config. 

I dont believe any breaking changes have been implemented to the backend since June 09, 2020 that would affect `V2.6.0` - so I expect it to function properly for you. Let us know if you have issues!

**Comment by hopewise** [comment]
Okey, thanks @ajrice6713
